include/univalue.h

```diff
@@ -73,6 +73,22 @@ class UniValue {
     bool checkObject(const std::map<std::string,UniValue::VType>& memberTypes) const;
     const UniValue& operator[](const std::string& key) const;
     const UniValue& operator[](size_t index) const;
+    const UniValue& at(const std::string& key) const {
+        return (*this)[key];
```

Comment:
In the std lib: "If pos is not within the range of the container, an exception of type std::out_of_range is thrown."

Should do the same here?

Reply:
I'm not sure. JSON typically treats `null` and absent as identical, so returning `null` (as this currently does) *might* make sense.

Reply:
> JSON typically treats `null` and absent as identical

Is this in the spec? Or is it implementation specific?

Reply:
I don't see it in any specs, but it is the current behaviour of UniValue's getters, and appears to be common among other JSON implementations as well.

Reply:
> Should do the same here?

I think it is fine as is.